migrations/20240517004955_keyword_handicap/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `keyword` ADD COLUMN `handicap` INTEGER NULL DEFAULT 0;
```

Comment:
I'm not able to spin this up from `main` without it causing issues, I think it has to do with capitalization of the table since we came across this before. Our table name is `Keyword` with a capital 'K', let's fix this to make sure we can start up our API repo without hiccups.

Reply:
I apologize, I forgot about that!!

Out of curiosity - is there a reason the migration file generates table names incorrectly? I'll write myself a note to check that on any future back end stories!

Reply:
Thanks! Unfortunately I'm not sure if there's a way to get keystone to get this right the first time around :/ I did some digging around their documentation but couldn't find anything obvious.